packages/mobile/ios/AudiusReactNative/AppDelegate.mm

```diff
@@ -263,4 +263,20 @@ - (Class)getModuleClassFromName:(const char *)name
 
 #endif
 
+- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url sourceApplication:(NSString *)sourceApplication annotation:(id)annotation
+{
+    if ([[TikTokOpenSDKApplicationDelegate sharedInstance] application:application openURL:url sourceApplication:sourceApplication annotation:annotation]) {
+        return YES;
+    }
+    return NO;
+}
```

Comment:
You can delete this - the logic is already in the existing `openURL` function above (I added it there for the Share Video to TikTok)

Reply:
Ok I'll try removing it, it wasn't working before I added it but I think I changed some other things at the same time

Reply:
Ooh gotcha